crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -575,17 +583,25 @@ pub fn check_visibility<QF>(
                 // If we have an aabb, do frustum culling
                 if !no_frustum_culling && !no_cpu_culling {
                     if let Some(model_aabb) = maybe_model_aabb {
+                        // Only use far culling if we have a projection with some `far` value.
+                        let use_far_culling = maybe_projection.is_some_and(|p| p.far().is_some());
+
                         let world_from_local = transform.affine();
                         let model_sphere = Sphere {
                             center: world_from_local.transform_point3a(model_aabb.center),
                             radius: transform.radius_vec3a(model_aabb.half_extents),
                         };
                         // Do quick sphere-based frustum culling
-                        if !frustum.intersects_sphere(&model_sphere, false) {
+                        if !frustum.intersects_sphere(&model_sphere, use_far_culling) {
```

Comment:
I thought this would preserve existing behaviour when using the default projections, but maybe I'm misunderstanding as `OrthographicProjection` previously far culled even though we always passed `false` here?